File: 1878-check-if-array-is-sorted-and-rotated/1878-check-if-array-is-sorted-and-rotated.java
class Solution {
    public boolean check(int[] nums) {
        int count=0;
        if(nums.length == 1){
            return true;
        }

        if(nums[nums.length-1] > nums[0]){
            count++;
        }

        for(int i=0;i<nums.length-1;i++){
            if(nums[i]>nums[i+1]){
                count++;
            }
        }
        
        return count>1?false:true;
    }
}

File: 1878-check-if-array-is-sorted-and-rotated/1878-check-if-array-is-sorted-and-rotated.py
class Solution(object):
    def check(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        if len(nums) == 1:
            return True
        
        count = 0
        if nums[len(nums)-1] > nums[0]:
            count +=1
        
        for i in range(len(nums)-1):
            if nums[i] > nums[i+1]:
                count += 1
        

        if count > 1:
            return False
        
        return True

        